skyportal/models/spectrum.py

```diff
@@ -385,8 +387,16 @@ def from_ascii(
         )
 
 
-SpectrumReducer = join_model("spectrum_reducers", Spectrum, User)
-SpectrumObserver = join_model("spectrum_observers", Spectrum, User)
+SpectrumReducer = join_model(
```

Comment:
This is clearly a big improvement.

Reply:
Yeah. I'm really surprised this thing even worked until now. The two relationships were overlapping and probably one killed the other.

Reply:
Yeah...